src/ManageLiquidity/ManageLiquidity.js

```diff
@@ -69,19 +69,29 @@ export default function ManageLiquidity(props) {
   const providePrice = useMemo(() => {
     return providePoolExists ? provideLiquidityToken.price : initialPrice;
   }, [provideLiquidityToken, providePoolExists, initialPrice]);
-  const removePrice = useMemo(() => {
-    return removeLiquidityToken.price;
-  }, [removeLiquidityToken]);
   const provideBaseTokenAmount = useMemo(
     () => (provideAmount * providePrice) / BASE_FACTOR,
     [provideAmount, providePrice]
   );
   const userProvideTokenBalance = useMemo(() => {
     return userTokens.filter((t) => t.id === provideToken.id)[0].balance;
   }, [provideToken, userTokens]);
-  const userRemoveTokenBalance = useMemo(() => {
-    return liquidityTokens.filter((t) => t.id === removeToken.id)[0].balance;
-  }, [removeToken, liquidityTokens]);
+
+  const userTokensInPool = useMemo(() => {
+    return (
+      Math.round(
```

Comment:
Note: this is to fix the issue where the pool is 4.5 tokens and your share is .444444444. Otherwise, it would display that you have 1.99999998 tokens, when it should just be 2.

Reply:
Ideally we stay in  `BigNum`s as long as possible to prevent overflow but I think we'll probably be fine here...